assets/README.md

```diff
@@ -1,4 +1,4 @@
-# Assets loaded in with IPFS
+# Assets loaded in with BTFS
 
 ## Generating docs
 
```

Comment:
Have you tried to see if the generating docs tool is generating useful docs?

Reply:
Robin and I both tried it, and while the command works, we can't seem to find the docs that are apparently being generated.

Reply:
I just deleted this Readme until we've figured it out.